pkgdb2/templates/packager.html

```diff
@@ -11,9 +11,9 @@
   {{ packager | avatar | safe }}
   {{ packager }}</h1> (<a class="fas" target="_blank"
     {% if packager.startswith('group::') %}
-        href="https://admin.fedoraproject.org/accounts/group/view/{{ packager.replace('group::', '') }}"
+        href="{{ config['PKGDB2_FAS_URL'] }}/group/view/{{ packager.replace('group::', '') }}"
```

Comment:
Is it not required to update `PKGDB2_FAS_URL` in `default_config.py` ? It is set to `None` right now.